com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -182,37 +182,16 @@ internal bool IsTeleportingNextFrame
             internal float ScaleX, ScaleY, ScaleZ;
             internal double SentTime;
 
-            internal Vector3 Position
-            {
-                get { return new Vector3(PositionX, PositionY, PositionZ); }
-                set
-                {
-                    PositionX = value.x;
-                    PositionY = value.y;
-                    PositionZ = value.z;
-                }
-            }
-
-            internal Vector3 Rotation
-            {
-                get { return new Vector3(RotAngleX, RotAngleY, RotAngleZ); }
-                set
-                {
-                    RotAngleX = value.x;
-                    RotAngleY = value.y;
-                    RotAngleZ = value.z;
-                }
-            }
+            internal bool IsDirty;
 
-            internal Vector3 Scale
+            /// <summary>
+            /// This will reset the NetworkTransform BitSet
+            /// </summary>
+            internal void ClearBitSetForNextTick()
             {
-                get { return new Vector3(ScaleX, ScaleY, ScaleZ); }
-                set
-                {
-                    ScaleX = value.x;
-                    ScaleY = value.y;
-                    ScaleZ = value.z;
-                }
+                var localSpace = (ushort)(m_Bitset & (1 << k_InLocalSpaceBit));
```

Comment:
```suggestion
                var localSpace = (ushort)(m_Bitset & (1 << k_InLocalSpaceBit)); // since we store the actual value (and not a dirty bit) here, we need to preserve it
```

Reply:
Good call.